modules/validations-before-agent-iso-creation.adoc

```diff
@@ -4,32 +4,33 @@
 
 :_content-type: REFERENCE
 [id="validations-before-agent-iso-creation_{context}"]
-= Validation checks before agent.iso creation
+= Validation checks before agent ISO creation
 
-The Agent-based Installer performs validation checks on user defined YAML files before the ISO is created. Once the validations are successfull, the `agent.iso`
+The Agent-based Installer performs validation checks on user defined YAML files before the ISO is created. Once the validations are successful, the agent ISO
 is created.
 
 .`install-config.yaml`
 
-* `baremetal`, `vSphere` and `none` platforms are supported.
-* If `none` is used as a platform, the number of control planes must be `1` and the total number of worker replicas must be `0`.
-* `apiVIPs` and `ingressVIPs` parameters must be set for bare metal and vSphere platforms respectively.
-* The `networkType` parameter must have a `OvNKubernetes` value.
+* `baremetal`, `vsphere` and `none` platforms are supported.
+* If `none` is used as a platform, the number of control plane replicas must be `1` and the total number of worker replicas must be `0`.
+* `apiVIPs` and `ingressVIPs` parameters must be set for bare metal and vSphere platforms.
+* The `networkType` parameter must be `OVNKubernetes` in the case of `none` platform.
```

Comment:
Ideally we would move this up a line so that the bullet about baremetal/vsphere platforms is not between the 2 about none platform

Reply:
ok make sense